src/main.c

```diff
@@ -2062,9 +2068,7 @@ int WINAPI WinMain(HINSTANCE hInst, HINSTANCE hPrev, LPSTR cmdline, int show)
 	return (int)msg.wParam;
 }
 
-#if defined(CONSOLE)
-void main(void) { WinMain(0, 0, 0, 0); }
-#endif
+int main(void) { return WinMain(0, 0, 0, 0); }
```

Comment:
When using MSVC or MinGW it selects the correct main function at link time. There is no reason to hide the definition.